File: EasyGroceries.Server/Models/Order.cs
namespace EasyGroceries.Server.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public int CustomerId { get; set; }
        public DateTime OrderDate { get; set; }
        public List<CartItem> CartItems { get; set; } = new();
        public decimal TotalAmount { get; set; }
        public bool HasLoyaltyMembership { get; set; }
        public ShippingInfo ShippingDetails { get; set; }
    }
}

File: easygroceries.client/src/models/Order.ts
export interface CartItem {
    productId: number;
    productName: string;
    description: string;
    quantity: number;
    unitPrice: number;
    totalPrice: number;
}

export interface ShippingInfo {
    name: string;
    address: string;
    city: string;
    pinCode: string;
    mobileNo: string;
}

export interface Order {
    items: CartItem[];
    hasLoyaltyMembership: boolean;
    shippingInfo: ShippingInfo;
}

export interface ShippingSlip {
    customerId: number;
    orderNumber: number;
    cartItems: CartItem[];
    shippingInfo: ShippingInfo;
}